dataactcore/models/domainModels.py

```diff
@@ -33,6 +38,23 @@ def concat_tas_dict(tas_dict):
     return tas
 
 
+def concat_display_tas_dict(tas_dict):
+    """ Given a dictionary, create a concatenated display TAS string. Copied directly from USASpending.gov. """
+    tas_rendering_label = "-".join(filter(None, (tas_dict['allocation_transfer_agency'],
+                                                 tas_dict['agency_identifier'])))
+
+    typecode = tas_dict['availability_type_code']
+    if typecode is not None and typecode != "":
```

Comment:
Can we just do `if typecode:`? I think it'll cover all these bases

Reply:
Agreed.